backend/app/routers/thumbnails.py

```diff
@@ -72,7 +73,9 @@ async def remove_thumbnail(thumb_id: str, fs: Minio = Depends(dependencies.get_f
 
 @router.get("/{thumbnail_id}")
 async def download_thumbnail(
-    thumbnail_id: str, fs: Minio = Depends(dependencies.get_fs)
+    thumbnail_id: str,
+    fs: Minio = Depends(dependencies.get_fs),
```

Comment:
Is minio is being used?

Reply:
Yes, it was used before. What I changed here is I added a downloads since to match the other fields in elasticsearch.

Reply:
Also - the downloads are not incrementing because I was not counting them as downloading if they are just displayed. Wasn't sure if I did that right.